manifests/course/puppetize.pp

```diff
@@ -17,8 +17,16 @@
     include classroom::master::dependencies::dashboard
     include classroom::master::hiera
 
+    $base_plugin_list = [ "Certificates", "Classification", "ConsoleUser", "Docker", "Logs", "Dashboard", "CodeManager", "ShellUser", "Gitviz" ]
```

Comment:
it'd be nice to figure out a good way to generalize this instead of copypasta. Part of the grand refactoring that will happen someday soon, I suppose.